src/styles/stars.scss

```
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+  width: 16px;
+  height: 16px;
+  background: url(../images/star.svg) no-repeat;
+  background-position: center;
+
+  &:not(:first-child) {
+  margin-left: 4px;
+  }
+  }
```

Pay attention on indents
```suggestion
  &__star {
    width: 16px;
    height: 16px;
    background: url(../images/star.svg) no-repeat;
    background-position: center;

    &:not(:first-child) {
      margin-left: 4px;
    }
  }
```